resolwe_bio/processes/slamdunk/alleyoop_summary.py

```diff
@@ -0,0 +1,90 @@
+"""Run Alleyoop summary tool on Slamdunk results."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, ListField, Process, StringField
+
+
+class AlleyoopSummary(Process):
+    """Run Alleyoop summary."""
+
+    slug = 'alleyoop-summary'
+    process_type = 'data:alleyoop:summary'
+    name = 'Alleyoop summary'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    data_name = '{{ slamdunk|sample_name|default("?") }}'
```

Comment:
Slamdunk input is a ListField. data_name should thus be `data_name = '{{ slamdunk.0|sample_name|default("?") }}'`

Reply:
This has been resolved, should be marked as such.